### ISSUE TITLE
Maven --batch-mode not supressing jib progress report 

### ISSUE THREAD

**Issue by fralalonde** [GoogleContainerTools/jib#1513]
When captured by non-interactive terminal, Jib's maven plugin output of transfer progress makes job logs much longer / dirtier than they should be. This is a minor, but very real pain point when trying to keep Continuous Integration logs meaningful and readable.

Maven provides the `--batch-mode` CLI option to suppress such progress reports. Jib should use the same mechanism to suppress it's output when specified. 

Here's a _partial_ sample output of jib when running in non-interactive terminal:

```
...
[INFO] Executing tasks:
[INFO] [==================            ] 60.7% complete
[INFO] > building dependencies layer
[INFO] > setting up to push layers
[INFO] > setting up to push layers
[INFO] > pulling base image layer sha256:6ac6f91f019c1...
[INFO] 
[INFO] Executing tasks:
[INFO] [==================            ] 60.8% complete
[INFO] > building dependencies layer
[INFO] > setting up to push layers
[INFO] > setting up to push layers
[INFO] > pulling base image layer sha256:6ac6f91f019c1...
[INFO] 
[INFO] Executing tasks:
[INFO] [==================            ] 60.9% complete
[INFO] > building dependencies layer
[INFO] > setting up to push layers
[INFO] > setting up to push layers
[INFO] > pulling base image layer sha256:6ac6f91f019c1...
[INFO] 
[INFO] Executing tasks:
[INFO] [==================            ] 61.0% complete
...
```

**Comment by chanseokoh** [comment]
Thanks for the PR. I also suggested in the past the batch mode should suppress the progress. Perhaps we should seriously consider it.

For now, you can set `-Djib.console=plain` to suppress the progress.

**Comment by fralalonde** [comment]
Thanks, I'll try that, still better than filtering.

**Comment by chanseokoh** [comment]
The previous discussion around disabling the progress for the batch mode: https://github.com/GoogleContainerTools/jib/pull/1344#discussion_r241636041

**Comment by chanseokoh** [comment]
Fixed by #1514.

**Comment by chanseokoh** [comment]
@fralalonde v1.0.2 released with the fix. Thanks for your contribution!
